### DISCUSSION TITLE
Can't add new student roster

### DISCUSSION THREAD

**Discussion by Krissiw** [community/community#17733] [GitHub Education]
[classroom] Tried uploading a file and directly pasting in a list of students' email address, keep getting an error saying student roster cannot be empty.
If I try to add a student to an existing roster it says 'No new student added' Help Please!

**Comment by samus-aran** [comment]
Hey @Krissiw How did you get on? Did the [Add the student roster documentation](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/manage-classrooms#adding-students-to-the-roster-for-your-classroom) help?

**Reply by Krissiw** [reply]
I got around it by just adding a list of student identifiers as opposed to email addresses. It is back to allowing me to upload a list of email addresses now (which was the previous behavior)
